web/src/views/TikTokDownloaderView.vue

```diff
@@ -1,249 +1,250 @@
-                    <div v-if="tikTokMediaData && tikTokMetaData && !isLoading && !error">
-                        <div class="card text-bg-dark mb-3 mt-5" >
-                            <div class="row g-0">
-                                <div class="col-md-4">
-                                    <img
-                                        class="img-fluid rounded-start"
-                                        :src="tikTokMetaData.thumbnail" />
-                                </div>
-
-                                <div class="col-md-8">
-                                    <div class="card-header">
-                                        <h5 class="card-title">
-                                            <i class="fa-solid fa-user"></i>
-                                            {{ tikTokMetaData.creator }}
-                                        </h5>
-                                    </div>
-
-                                    <div class="card-body">
-                                        <p class="card-text">
-                                            {{ tikTokMetaData.description }}
-                                        </p>
-                                    </div>
-
-                                    <div class="card-body">
-                                        <ul class="list-group list-group-flush">
-                                            <li
-                                                v-for="(item, index) in tikTokMediaData"
-                                                class="list-group-item thumbnail"                                                              
-                                                :key="index"
-                                                @click="downloadVideo(item.url)">
-                                                <i class="fa-solid fa-circle-arrow-down"></i>
-
-                                                <span>
-                                                    Download {{ item.formatNote }} [{{ Number(index) + 1 }}]
-                                                </span>
-                                            </li>
-                                        </ul>
-                                    </div>
+	<div class="container">
+		<div
+			class="d-flex flex-column justify-content-center align-items-center">
+			<lottie-player
+				v-if="!tikTokMediaData"
+				src="https://assets7.lottiefiles.com/packages/lf20_5h2kp8uz.json"
+				background="transparent"
+				speed="0.5"
+				style="width: 300px; height: 300px;"
+				loop
+				autoplay />
+
+			<div class="col-lg-8 text-center">
+				<figure 
+					v-if="!tikTokMediaData && !formError"
+					class="text-center">
+					<blockquote class="blockquote">
+						<h3 class="mb-4">
+							TikTok Downloader
+						</h3>
+					</blockquote>
+
+					<figcaption class="blockquote-footer">
+						Download TikTok Videos in seconds...
+					</figcaption>
+				</figure>
```

Comment:
Can we follow the [QR Code design](https://github.com/pizza-society/unicornify/blob/PS-59-Refactor-Web/web/src/views/QRGeneratorView.vue#L15-L22) for this section? Or you feel it look better like this?